bower.json

```diff
@@ -33,7 +33,7 @@
   ],
   "dependencies": {
     "angular": "1.6.9",
-    "rico-js": "https://github.com/rico-project/rico-js.git#master",
+    "rico-js": "1.0.0-CR.3",
```

Comment:
as rico.js is not published to npm, I expect this to fail. 
Shouldn't this be somthing like "https://github.com/rico-project/rico-js/tree/1.0.0-CR.3" ?

Reply:
No. Bower will fetch the correct version because of the tags aka releases on github.

Reply:
https://github.com/rico-project/rico-js/releases